saturn-datepicker/src/datepicker/month-view.ts

```diff
@@ -89,10 +89,10 @@ export class SatMonthView<D> implements AfterContentInit {
   _rangeFull: boolean | null = false;
 
   /** Whenever user already selected start of dates interval. */
-  @Input() set beginDateSelected(value: D | false) { this._beginDateSelected = !!value } ;
+  @Input() set beginDateSelected(value: D | false) { this._beginDateSelected = value } ;
 
   /** Whenever user already selected start of dates interval. An inner property that avoid asynchronous problems */
-  private _beginDateSelected: boolean = false;
+  _beginDateSelected: D | boolean = false;
```

Comment:
```suggestion
  _beginDateSelected: D | null;
```
I think null is more homogeneous for code. (since we are not using `true` anymore)

Reply:
https://github.com/SaturnTeam/saturn-datepicker/pull/68/commits/1548cc6383a0017872bdb95a4431a428360bd04b

Reply:
& https://github.com/SaturnTeam/saturn-datepicker/pull/68/commits/edb674d49b2a7665408ccf9469521929e6ff89d8